src/WKWebKit/Defs.cs

```diff
@@ -113,4 +113,28 @@ public enum WKAudiovisualMediaTypes : ulong	{
 		Video = 1 << 1,
 		All = UInt64.MaxValue
 	}
+
+	[Mac (11,3), iOS (14,5), MacCatalyst(14,5)]
+	[Native]
+	public enum WKMediaCaptureState : long {
+		None,
+		Active,
+		Muted,
+	}
+
+	[Mac (11,3), iOS (14,5), MacCatalyst (14,5)]
```

Comment:
I'm always suspicious about retroactive headers...
The bots will be testing this on macOS 11.3 (and Catalyst 14.5)
Did you run intro on a iOS 14.5 simulator (or device) ?

Reply:
I have the same feeling, I started with all changes to be iOS (15,0) against the header info, but though leaving it for the review